target/linux/ipq40xx/base-files/etc/board.d/02_network

```diff
@@ -138,12 +138,6 @@ ipq40xx_setup_macs()
 	8dev,habanero-dvk)
 		label_mac=$(mtd_get_mac_binary "ART" 0x1006)
 		;;
-	asus,rt-ac58u)
-		CI_UBIPART=UBI_DEV
-		wan_mac=$(mtd_get_mac_binary_ubi Factory 0x1006)
-		lan_mac=$(mtd_get_mac_binary_ubi Factory 0x5006)
```

Comment:
Well, you cant just drop the MAC setting.
You can use NVMEM for getting the MAC directly in the DTS